frontend/src/lib/components/ActivationSidebar/activationLogic.ts

```diff
@@ -319,21 +310,19 @@ export const activationLogic = kea<activationLogicType>([
             )
         },
     })),
-    events(({ actions, values }) => ({
+    events(({ actions }) => ({
         afterMount: () => {
-            if (values.shouldShowSecondaryOnboarding) {
-                // we artificially wait for a second so that the UI has time to render before we check for these async values
-                // this prevents the UI from flickering when the values are loaded
-                setTimeout(() => {
-                    actions.loadCustomEvents()
-                    actions.loadInsights()
-                }, 1000)
-            }
+            // we artificially wait for a second so that the UI has time to render before we check for these async values
+            // this prevents the UI from flickering when the values are loaded
+            setTimeout(() => {
+                actions.loadCustomEvents()
+                actions.loadInsights()
+            }, 1000)
```

Comment:
I'm not certain if there is a good solution here but I generally try to avoid using timeouts for waiting for things to happen as they can be unreliable if, for instance, someone is on a slow connection. I don't think it matters much here since a slight flicker is not a big deal but figured I'd mention it in case you had an idea for making it more robust. Eg. some event we can listen for that tells us that the UI is ready.